ivy/functional/backends/paddle/elementwise.py

```diff
@@ -377,14 +377,12 @@ def cos(x: paddle.Tensor, /, *, out: Optional[paddle.Tensor] = None) -> paddle.T
     return paddle.cos(x)
 
 
-@with_unsupported_dtypes({"2.6.0 and below": ("uint8", "float16")}, backend_version)
+@with_unsupported_dtypes(
+    {"2.6.0 and below": ("uint8", "float16", "bfloat16")}, backend_version
+)
 def logical_not(
     x: paddle.Tensor, /, *, out: Optional[paddle.Tensor] = None
 ) -> paddle.Tensor:
-    if paddle.is_complex(x):
```

Comment:
we seem to be removing the complex dtype handling. So is complex unsupported now or does it still work?

Reply:
yes it's working bcz the paddle.logical_not can handle the complex dtype also.  [here](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/logical_not_en.html) the paddle doc for this and i have tested this passing all tests in local.